pandas/compat/__init__.py

```diff
@@ -26,7 +26,6 @@
 from pandas.compat.numpy import is_numpy_dev
 from pandas.compat.pyarrow import (
     HAS_PYARROW,
-    pa_version_under10p1,
     pa_version_under11p0,
```

Comment:
pa_version_under11p0 can be removed also?

Reply:
Yes, hadn't yet done any cleanup of compat code, just the minimal changes to bump the version. The last added commit now removes all usages of pa_version_under11p0